src/main/java/com/nutomic/syncthingandroid/syncthing/RestApi.java

```diff
@@ -153,6 +153,8 @@ public void setParams(int keep) {
 
     private String mApiKey;
 
+    private String mHttpsCertPath;
```

Comment:
You can make this final (`mApiKey` and `mOnApiAvailableListener` too, actually).